demoapp/docker-compose-fem-demoapp.yml

```diff
@@ -6,7 +6,8 @@ networks:
 services:
   fem_demoapp:
```

Comment:
@ronnyfriedland , I think the definition of the project name is needed now, e.g. add: "--project-name fem".
With the separation into modules it must be ensure that all components run in the same network.
As you know without configure the project name, the name for the container as well as the name for the network were created from the directory name by default. That results into different subnets for the containers. The avoid that we have to set the project name and remove the "fem_"-prefix in all docker-compose files.